Libraries/Opc.Ua.Server/Configuration/ConfigurationNodeManager.cs

```diff
@@ -160,6 +160,14 @@ ApplicationConfiguration configuration
                             {
                                 passiveNode.Parent.ReplaceChild(context, activeNode);
                             }
+                            else
+                            {
```

Comment:
why is the servernode patched here?

Reply:
The ServerConfiguration node that is created at this step does not have the Server node set as a Parent and this does not allow the AuditEvents raised by the nodes under the ServerConfiguration (e.g. The TrustList objects in the CertificateGroups, The UpdateCertificate method) node to bubble up to the Server node.